root/files/resource/Sql/oracle/Create_UserStore.sql

```diff
@@ -0,0 +1,112 @@
+-- see : Oracle11gXE + ODP.NET Managed Driver - マイクロソフト系技術情報 Wiki

```

Comment:
@daisukenishino 

Please indicate the reasons for seeing this site.

For example:  
For the information of using Oracle database and ODP.NET managed driver for the user store of multi-purpose authentication site, see the following site.